src/Compatibility/Core/src/iOS/Extensions/FontExtensions.cs

```diff
@@ -5,11 +5,11 @@ namespace Microsoft.Maui.Controls.Compatibility.Platform.iOS
 {
 	public static partial class FontExtensions
 	{
-		public static UIFont ToUIFont(this Font self)
+		public static UIFont ToUIFont(this Maui.Font self)
```

Comment:
Should these be `ToNative` ?

Reply:
They're already in use in Controls renderers. And since they're public, they may already be in use by custom/3rd party renderers. So we can add a ToNative override, but these would still need to be here.